proto/pluginv2.proto

```diff
@@ -7,44 +7,59 @@ option go_package = ".;pluginv2";
 // Common
 //-----------------------------------------------
 
-message DataSourceConfig {
+message AppInstanceSettings {
+  bytes jsonData = 3;
+  map<string,string> decryptedSecureJsonData = 4;
+  int64 lastUpdatedMS = 5;
+}
+
+message DataSourceInstanceSettings {
   int64 id = 1;
   string name = 2;
   string url = 3;
   string user = 4;
   string database = 5;
   bool basicAuthEnabled = 6;
   string basicAuthUser = 7;
-
-  // from [data_source.json_data] field in the database
   bytes jsonData = 8;
-
-  // from [data_source.secure_json_data] field in the database
   map<string,string> decryptedSecureJsonData = 9;
   int64 lastUpdatedMS = 10;
 }
 
-message PluginConfig {
-  int64 orgId = 1;
-  string pluginId = 2;
-
-  // from [plugin_setting.json_data] field in the database
-  bytes jsonData = 3;
-
-  // from [plugin_setting.secure_json_data] field in the database
-  map<string,string> decryptedSecureJsonData = 4;
-  int64 lastUpdatedMS = 5;
-
-  DataSourceConfig datasourceConfig = 6;
-}
-
 message User {
   string login = 1;
   string name = 2;
   string email = 3;
   string role = 4;
 }
 
+message PluginContext {
+  // The Grafana organization id the request originating from.
```

Comment:
```suggestion
  // The Grafana organization id the request originates from.
```

Reply:
This should be changed in https://github.com/grafana/grafana-plugin-sdk-go/blob/master/proto/backend.proto

Reply:
@marefr OK thanks for the heads up.